diff --git a/Agda.cabal b/Agda.cabal
index b7cdd27..426351d 100644
--- a/Agda.cabal
+++ b/Agda.cabal
@@ -864,9 +864,12 @@ executable agda
   -- been idle for 0.3 s. This feature turned out to be annoying, so
   -- the idle GC is now by default turned off (-I0).
   ghc-options:
-    -threaded
+    -- -threaded
     -rtsopts
     -with-rtsopts=-I0
+    -fPIC
+  ld-options:
+    -static
 
 -- agda-mode executable
 ---------------------------------------------------------------------------
@@ -882,3 +885,7 @@ executable agda-mode
     , filepath  >= 1.4.2.1  && < 1.5
     , process   >= 1.6.3.0  && < 1.7
   default-language: Haskell2010
+  ghc-options:
+    -fPIC
+  ld-options:
+    -static
diff --git a/src/full/Agda/Interaction/Library.hs b/src/full/Agda/Interaction/Library.hs
index 3b81319..19cdf32 100644
--- a/src/full/Agda/Interaction/Library.hs
+++ b/src/full/Agda/Interaction/Library.hs
@@ -127,10 +127,11 @@ mkLibM libs m = do
 getAgdaAppDir :: IO FilePath
 getAgdaAppDir = do
   -- System-specific command to build the path to ~/.agda (Unix) or %APPDATA%\agda (Win)
-  let agdaDir = getAppUserDataDirectory "agda" >>= \legacyAgdaDir ->
-        ifM (doesDirectoryExist legacyAgdaDir)
-            (pure legacyAgdaDir)
-            (getXdgDirectory XdgConfig "agda")
+  -- let agdaDir = getAppUserDataDirectory "agda" >>= \legacyAgdaDir ->
+  --       ifM (doesDirectoryExist legacyAgdaDir)
+  --           (pure legacyAgdaDir)
+  --           (getXdgDirectory XdgConfig "agda")
+  let agdaDir = return "/root/.agda"
   -- The default can be overwritten by setting the AGDA_DIR environment variable
   caseMaybeM (lookupEnv "AGDA_DIR") agdaDir $ \ dir ->
       ifM (doesDirectoryExist dir) (canonicalizePath dir) $ do
