diff --git a/Agda.cabal b/Agda.cabal
index eea0735..8a252df 100644
--- a/Agda.cabal
+++ b/Agda.cabal
@@ -869,6 +869,9 @@ executable agda
     -threaded
     -rtsopts
     -with-rtsopts=-I0
+    -fPIC
+  ld-options:
+    -static

 -- agda-mode executable
 ---------------------------------------------------------------------------
@@ -884,3 +887,7 @@ executable agda-mode
     , filepath  >= 1.4.2.1  && < 1.6
     , process   >= 1.6.3.0  && < 1.7
   default-language: Haskell2010
+  ghc-options:
+    -fPIC
+  ld-options:
+    -static
