diff --git a/Agda.cabal b/Agda.cabal
index eea0735..8a252df 100644
--- a/Agda.cabal
+++ b/Agda.cabal
@@ -869,6 +869,9 @@ executable agda
     -threaded
     -rtsopts
     -with-rtsopts=-I0
+    -fPIC
+  ld-options:
+    -static

 -- agda-mode executable
 ---------------------------------------------------------------------------
@@ -884,3 +887,7 @@ executable agda-mode
     , filepath  >= 1.4.2.1  && < 1.6
     , process   >= 1.6.3.0  && < 1.7
   default-language: Haskell2010
+  ghc-options:
+    -fPIC
+  ld-options:
+    -static
diff --git a/src/full/Agda/Interaction/Library.hs b/src/full/Agda/Interaction/Library.hs
index 3b81319..19cdf32 100644
--- a/src/full/Agda/Interaction/Library.hs
+++ b/src/full/Agda/Interaction/Library.hs
@@ -127,10 +127,11 @@ mkLibM libs m = do
 getAgdaAppDir :: IO FilePath
 getAgdaAppDir = do
   -- System-specific command to build the path to ~/.agda (Unix) or %APPDATA%\agda (Win)
-  let agdaDir = getAppUserDataDirectory "agda" >>= \legacyAgdaDir ->
-        ifM (doesDirectoryExist legacyAgdaDir)
-            (pure legacyAgdaDir)
-            (getXdgDirectory XdgConfig "agda")
+  -- let agdaDir = getAppUserDataDirectory "agda" >>= \legacyAgdaDir ->
+  --       ifM (doesDirectoryExist legacyAgdaDir)
+  --           (pure legacyAgdaDir)
+  --           (getXdgDirectory XdgConfig "agda")
+  let agdaDir = return "/root/.agda"
   -- The default can be overwritten by setting the AGDA_DIR environment variable
   caseMaybeM (lookupEnv "AGDA_DIR") agdaDir $ \ dir ->
       ifM (doesDirectoryExist dir) (canonicalizePath dir) $ do
